packages/Webkul/Admin/src/Resources/lang/en/app.php

```diff
@@ -912,7 +912,7 @@
             'create'          => 'Create',
             'add-title'       => 'Create Tax Category',
             'edit-title'      => 'Edit Tax Category',
-            'save-btn-title'  => 'Save Tax Category',
+            'save-btn-title'  => 'Save',
```

Comment:
If the translation is being changed, then make sure all files present in the lang folder should be present here.

Reply:
means we have to override all the lang files?